tests/test_calculated_data.py

```diff
@@ -104,19 +105,19 @@ class TestCalculatedArray(unittest.TestCase):
 
     def test_static(self):
         dataset = xr.Dataset()
-        array = CalculatedArray(dataset, "3 * 5")
+        array = CalculatedArray(dataset, "3 * 5", [])
         self.assertEqual(array[0], 15)
 
     def test_passthrough(self):
         dataset = xr.Dataset({'var': ('x', [1, 2, 3, 4, 5])})
-        array = CalculatedArray(dataset, "var")
+        array = CalculatedArray(dataset, "var", [])
```

Comment:
These dims can be empty lists since these tests are not considering the shape property.